src/components/organisms/AutoLogoutWarning.vue

```diff
@@ -94,15 +99,9 @@ export default {
 			}
 		},
 	},
-	created(ctx) {
-		this.$store.dispatch("autoLogout/init", null, { root: true });
-	},
-	beforeDestroy() {
-		//this.$store.dispatch("autoLogout/reset");
-	},
-	methods: {
-		extendSession() {
-			this.$store.dispatch("autoLogout/extendSession");
+	eventBus: {
+		"showToast@autologout": function (value) {
+			this.showToast(value);
```

Comment:
```suggestion
		"showToast@autologout": (value) => {
			this.showToast(value);
```
geht das nicht auch als arrow function? dann ist klarer, dass der this kontext von der Komponente weiterhin genutzt wird.

Reply:
This is not possible, otherwise `this` is undefined

Reply:
but it is an arrow function 🤔  But ok, I trust you :)